boa/environment.py

```diff
@@ -531,7 +142,7 @@ def lookup_alias(self, address):
 
     # advanced: reset warm/cold counters for addresses and storage
     def _reset_access_counters(self):
-        self.vm.state._account_db._reset_access_counters()
+        self.evm.reset_access_counters()
```

Comment:
out of scope, but this is related to transaction context. we should probably have an API which bundles all things related to txn context together (off the top of my head, that is tx.origin, transient storage and the warm lists aka access acounters)